### ISSUE TITLE
Changes to the Generated TS Schema

### ISSUE THREAD

**Issue by shaunpearsondev** [danielearwicker/flowerbi#39]
Greetings!

I am looking into building a UI for flowerbi to allow users to create their own reports.

Early prototype here: https://github.com/bigwhitesolutions/reporting

One of the weird things I am having to do is define a const with all tables into allow me to build this:

```
const SchemaValues = [
  ...Object.values(Product),
  ...Object.values(ProductSubcategory),
];
```

It would be great if the generated document was contained within a parent then allowing me to iterate over it eg:

```
export const ProductSubcategory = {
  ProductSubcategoryID: new QueryColumn<number>(
    "ProductSubcategory.ProductSubcategoryID"
  ),
  Name: new QueryColumn<string>("ProductSubcategory.Name"),
};

export const Product = {
  ProductId: new QueryColumn<number>("Product.ProductId"),
  Name: new QueryColumn<string>("Product.Name"),
  ProductSubcategoryID: new QueryColumn<number>("Product.ProductSubcategoryID"),
};

export const AdventureWorks2019 = {
  Product,
  ProductSubcategory
}
```
Allowing me to write:

`const QueryColumns = Object.values(AdventureWorks).flatMap(x => Object.values(x));`

What are your thoughts on adding this?

Thanks

Shaun

**Comment by danielearwicker** [comment]
Yep, great idea. I've done something very similar, the only effort is in making the designer UI look nice!

I found it useful to "curate" the schema to help the user, deciding which columns will be suitable for being categories, or aggregated values (same-currency amounts), or filters, so that means I'm going to be listing them out anyway with knowledge of the specifics of the schema.

But if the schema TS output was rich enough then it could be auto-generated. So as well as giving you a list of all the tables, it should have the column types as well.

**Comment by danielearwicker** [comment]
Released in 3.1.0
